css/src/components/layout.scss

```diff
@@ -0,0 +1,110 @@
+$side-bar-width-wide: 4fr !default;
```

Comment:
Haven't figured out yet why, but the outline color is off on the layout page, it's a light grey while should be black. Posting like that so I don't forget:

![image](https://github.com/microsoft/atlas-design/assets/57374379/5b048dd5-834b-4797-a8bf-5484fd4ead37)

Reply:
Looking!

Reply:
debug classes were messing with outline colors for no reason, fixed!